ocs_ci/ocs/resources/pod.py

```diff
@@ -878,6 +878,24 @@ def get_noobaa_operator_pod(
     return noobaa_operator_pod
 
 
+def get_noobaa_core_pod(
+    ocs_label=constants.NOOBAA_CORE_POD_LABEL,
+    namespace=config.ENV_DATA["cluster_namespace"],
+):
+    noobaa_core = get_pods_having_label(ocs_label, namespace)
+    noobaa_core_pod = Pod(**noobaa_core[0])
+    return noobaa_core_pod
+
+
+def get_noobaa_endpoint_pods(
+    ocs_label=constants.NOOBAA_ENDPOINT_POD_LABEL,
+    namespace=config.ENV_DATA["cluster_namespace"],
+):
+    noobaa_endpoints = get_pods_having_label(ocs_label, namespace)
+    noobaa_endpoint_pods = [Pod(**pod) for pod in noobaa_endpoints]
+    return noobaa_endpoint_pods
+
+
```

Comment:
Please add Docstrings - I would mainly be interested in the return types (instance/s of the Pod class) since they're different than the return type of `get_pods_having_label`.

Also, I suggest not using `ocs_label` as an argument in these methods because it allows this kind of confusing usage:
```
nb_db_pod = get_noobaa_core_pod(ocs_label=constants.NOOBAA_DB_LABEL_47_AND_ABOVE)
```